curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fab9f17fa294054b74228c.md

```diff
@@ -48,18 +48,27 @@ assert.equal(document.querySelector('fieldset:nth-child(3) > label > input')?.ty
     <p>Please fill out this form with the required information</p>
     <form action='https://register-demo.freecodecamp.org'>
       <fieldset>
-        <label>Enter Your First Name: <input type="text" required /></label>
-        <label>Enter Your Last Name: <input type="text" required /></label>
-        <label>Enter Your Email: <input type="email" required /></label>
-        <label>Create a New Password: <input type="password" pattern="[a-z0-5]{8,}" required /></label>
+        <label for="first-name">Enter Your First Name: <input id="first-name" type="text" required /></label>
+        <label for="last-name">Enter Your Last Name: <input id="last-name" type="text" required /></label>
+        <label for="email">Enter Your Email: <input id="email" type="email" required /></label>
+        <label for="new-password">Create a New Password: <input id="new-password" type="password" pattern="[a-z0-5]{8,}" required /></label>
       </fieldset>
       <fieldset>
+<<<<<<< HEAD
+        <label for="personal-account"><input id="personal-account" type="radio" name="account-type" /> Personal Account</label>
+        <label for="business-account"><input id="business-account" type="radio" name="account-type" /> Business Account</label>
+        <label for="terms-and-conditions">
+          <input id="terms-and-conditions" type="checkbox" required /> I accept the
+				    <a href="https://www.freecodecamp.org/news/terms-of-service/">terms and conditions</a>
+			  </label>
+=======
         <label><input type="radio" name="account-type" /> Personal Account</label>
         <label><input type="radio" name="account-type" /> Business Account</label>
         <label>
           <input type="checkbox" required /> I accept the
             <a href="https://www.freecodecamp.org/news/terms-of-service/">terms and conditions</a>
         </label>
+>>>>>>> 39b5842b79db5e82ce58b3c0c3e2c43452dde180
```

Comment:
Please go through the files and remove the merge conflict syntax
```suggestion
<<<<<<< HEAD
        <label for="personal-account"><input id="personal-account" type="radio" name="account-type" /> Personal Account</label>
        <label for="business-account"><input id="business-account" type="radio" name="account-type" /> Business Account</label>
        <label for="terms-and-conditions">
          <input id="terms-and-conditions" type="checkbox" required /> I accept the
				    <a href="https://www.freecodecamp.org/news/terms-of-service/">terms and conditions</a>
			  </label>
=======
        <label><input type="radio" name="account-type" /> Personal Account</label>
        <label><input type="radio" name="account-type" /> Business Account</label>
        <label>
          <input type="checkbox" required /> I accept the
            <a href="https://www.freecodecamp.org/news/terms-of-service/">terms and conditions</a>
        </label>
>>>>>>> 39b5842b79db5e82ce58b3c0c3e2c43452dde180
```

Please make sure to use the new indentation, and the attributes you have added